test/xml/RSS-PodcastIndex-R1-NoPodcastValue.xml

```diff
@@ -0,0 +1,135 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This is just a duplicate of the existing `RSS-PodcastIndex-R1` test data. The only difference is that I deleted the podast-level `value` tag to check if it would correctly parse to a `null` on podcast-level (previously it would've taken the first episode-level `value` tag which would've been wrong).